Vagrantfile

```diff
@@ -171,34 +193,13 @@ Vagrant.configure(2) do |config|
     mount_shares(hoot_centos7)
 
     # We do want to add repos and update this box
+    # NOTE: This change applies to every target AFTER this one as well.
```

Comment:
This messed me up for a while. I could not figure out why the ENV vars kept getting stomped on when I had the additional targets added AFTER this target